unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarEditorHUD/Scripts/CharacterPreviewController.cs

```diff
@@ -63,6 +70,19 @@ private void Awake()
             new SimpleGPUSkinning(),
             new GPUSkinningThrottler()
         );
+        mouseWheelAction.OnValueChanged += OnMouseWheelChangeValue;
```

Comment:
We shouldn't be unsubscribing from this event on the OnDestroy()?

Reply:
Done